src/global.less

```diff
@@ -29,3 +29,7 @@ body {
     margin: 40px 0 !important;
   }
 }
+
+.hide {
```

Comment:
Is this CSS being referenced somewhere? I didn't find any reference to it in markup.

Reply:
@aquibbaig ?

Reply:
Hey @gurbirkalsi. I use this css as a className in our table component definition to hide the favorites column for non-registered users

Reply:
We might want to consider conditionally appending the `actions` column if the user is logged in during the render process. In general, CSS defined in `global.less` should be reserved for defining base layout/fonts.

Reply:
We can remove this CSS now that separate .less files have been created for each page component.